tests/eval/ooo_wb/ooo_wb.S

```diff
@@ -0,0 +1,37 @@
+.text
+.balign 16
+.globl test
+
+test:
+    la t1, test_src_start  # t1 = addr
+    la t2, test_src_end
+    sub t2, t2, t1  # t2 = remaining length
+    xor t3, t3, t3  # t3 = sum
+loop_start:
+    vsetvli t0, t2, e8, m8, ta, ma   # t0 = vl
+    vle8.v v8, (t1)
+    vadd.vi v8, v8, 1
+
+    # scaler workload
+    lw t4, 0(t1)
```

Comment:
`lw t4, 0(t1)` will block `vle8.v v8, (t1)`

Reply:
V-V(mem)
indexed load to `v1`, then execute some how vector instructions not depending on `v1`, then use `v1` for the final exec, and start another round of indexed load

V-V(exec)
younger instr can retire before older instr, i haven't come up with a good example...

V-S
can schedule non-dependent scalar code in a large convoy of vector instructions, e.g. vld, add, vadd, sub, vmul, mul ...